tests/tap.sh

```diff
@@ -62,6 +62,15 @@ for ((i=0;i<${#backends[@]}; ++i)); do
         continue
     fi
 
+    # grep to pass test t308 on error
```

Comment:
We're good now, but as our test suite grows I think we'll have to occasionally check that our exceptions in here still uniquely identify error messages to make sure that 'expected failures' happened the right way.